# Notes on Computational Genomics with R
#### by Altuna Akalin


This is somewhat an opinionated guide on using R for computational genomics. It is aimed at wet-lab researchers who wants to use R in their data analysis ,and bioinformaticians who are new to R and wants to learn more about its capabilities for genomics data analysis.

As new high-throughout experimental techniques on the rise, data analysis capabilities are sought-after features for researchers. R, with its statistical heritage, plotting features and rich user-contributed packages is one of the best languages for the task of analyzing data. The book gives a brief introduction on basics of R and later divided to chapters that represent subsets of genomics data analysis.

### What will you get out of this
This resource describes the skills and provides how-tos that will help readers analyze their own genomics data.

After reading:
* If you are not familiar with R, you will get the basics of R and divide right in to specialized uses of R for computational genomics.
* you will understand genomic intervals and operations on them, such as overlap
* You will be able to use R and its vast package library to do sequence analysis: Such as calculating GC content for given segments of a genome or find transcription factor binding sites
* You will be familiar with visualization techniques used in genomics, such as heatmaps,meta-gene plots and genomic track visualization
* You will be familiar with supervised and unsupervised learning techniques which are important in data modelling and exploratory analysis of high-dimensional data



### Contribute to the development
You can contribute to the development of this guide using github features such as pull-requests and issue creation.




